src/dom.js

```diff
@@ -240,6 +240,17 @@ export function isShadowRoot(value) {
   );
 }
 
+/**
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function isInFie(element) {
+  return (
+    element.classList.contains('i-amphtml-fie') ||
+    !!closestAncestorElementBySelector(element, '.i-amphtml-fie')
+  );
+}
```

Comment:
I'd move this to iframe-helper.js too. Keeps FIE functions together.

Reply:
SG done